### DISCUSSION TITLE
Are MOC's (Maps of Content) useful? should i forgo a structured folder hierarchy

### DISCUSSION THREAD

**Discussion by liutiming** [BryanJenksCommunity/FAQ#14] [Q&A] [answered]
### Question

What is your question?

Is MOC useful? You once said that focus on ideas, and not graphs. It seems to be quite some effort to build MOC and organising thoughts into specific categories seem to contradict the advantage of obsidian over other more organised note taking apps like OneNote. Overall, I feel it can quite time-consuming to catch up with recent development of note-taking techniques and distracting of taking notes itself. 

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
I think MOC's are incredibly useful and have made use of them since the first days of my vault. 

They become essential as the vault grows when you lack a structured folder hierarchy. Ultimately every case for each individual is different, i chose the approach of a lack of structured heirarchy due to the power of linking, MOC's, and search.  I have no need for folders and can find everything i need this way. 

The once a vault starts reaching critical mass, it will be hard to find/stumble upon various items if following a link path, this is why MOC's are useful. Not necessary to index EVERY note, but to over time, identify large clusters of topics/ideas/discussion that are diving boards. It's less of a list of every topin related to [[Productivity]] and more of a list of [[Productivity Applications]], [[Productivity Mindsets]] which branch off into their own link trees

constantly configuring the system and spending more time building the system and not using it is procrastiplanning or procrastibation. The system is meant to be used, honestly MOC's shouldn't be made and rigidly enfored at all until your vault becomes too big to be easily managed without them. Let the structure emerge naturally, don't set up a vault on day 1 and simultaneous make a whole list of all of your MOC's on that day without any notes in the vault. That leads to you then wondering where each note should be placed after you write it and spending more time focusing on organization then notating. 

Write notes, See structure emerge (larger nodes on the graph based on link count) promote those notes to MOC's and make those jumping off points into your topic clusters.

great accessory explanation is how i approach tagging to facilitate some of this: #12
